src/libraries/System.Private.CoreLib/src/System/Numerics/DivisionRounding.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Numerics
+{
+    /// <summary>
+    /// Specifies the rounding strategy to use when performing division.
+    /// </summary>
+    public enum DivisionRounding
+    {
+        /// <summary>
+        /// Truncated division (rounding toward zero) — round the division result towards zero.
+        /// Graph for truncated division with positive divisor https://www.wolframalpha.com/input?i=Plot%5B%7BIntegerPart%5Bn%5D%2C+n+-+IntegerPart%5Bn%5D%7D%2C+%7Bn%2C+-3%2C+3%7D%5D
+        /// Graph for truncated division with negative divisor https://www.wolframalpha.com/input?i=Plot%5B%7BIntegerPart%5B-n%5D%2C+n+%2B+IntegerPart%5B-n%5D%7D%2C+%7Bn%2C+-3%2C+3%7D%5D
```

Comment:
We probably don't want to reference external source in our shipping documentation.

Reply:
How can I create internal graphs?

Reply:
Adding graph to api doc can be non-trivial work at https://github.com/dotnet/dotnet-api-docs or https://github.com/dotnet/docs.

Adding external link into code-only comments should be fine if it doesn't violate any license, but they shouldn't be in xml doc.